ports/rp2/rp2_flash.c

```diff
@@ -120,6 +120,7 @@ STATIC mp_obj_t rp2_flash_readblocks(size_t n_args, const mp_obj_t *args) {
         offset += mp_obj_get_int(args[3]);
     }
     memcpy(bufinfo.buf, (void *)(XIP_BASE + self->flash_base + offset), bufinfo.len);
+    MICROPY_EVENT_POLL_HOOK_FAST;
```

Comment:
It would be good to add a comment about why this is here.  So it can be eventually removed if TinyUSB processing is done at IRQ time.

Reply:
Thanks for the comments. I'll do that.